modules/swagger-codegen/src/main/resources/typescript-angular/modelGeneric.mustache

```diff
@@ -16,13 +16,8 @@ export interface {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{
 export namespace {{classname}} {
 {{#vars}}
     {{#isEnum}}
-    export enum {{enumName}} {
-    {{#allowableValues}}
-        {{#enumVars}}
-        {{{name}}} = <any> {{{value}}}{{^-last}},{{/-last}}
-        {{/enumVars}}
-    {{/allowableValues}}
-    }
+    export type {{enumName}} = {{#allowableValues}}{{#enumVars}}{{{value}}}{{^-last}} | {{/-last}}{{/enumVars}}{{/allowableValues}};
     {{/isEnum}}
 {{/vars}}
-}{{/hasEnums}}
\ No newline at end of file
+}{{/hasEnums}}i
```

Comment:
I'm afraid you accidentally inserted an `i` there 🙈

Reply:
Oops, indeed. Fixed.